### DISCUSSION TITLE
problem with deploying contract on zksync

### DISCUSSION THREAD

**Discussion by satyaorz** [Cyfrin/foundry-full-course-cu#2797] [Q&A]
it says "Contracts folder not found in the workspace" so i am unable to compile the solidity contract file.
screenshots:

![Screenshot From 2024-11-03 01-18-04](https://github.com/user-attachments/assets/2041fe1d-5164-44ee-b8b9-9f1cc101bd9d)
 
![Screenshot From 2024-11-03 01-12-10](https://github.com/user-attachments/assets/c05d4981-bd47-4237-befc-aa321c53cf31)
how do i fix this, or i can skip this? as i see many discussion saying we may skip deploying on zksync.

**Comment by satyaorz** [comment]
it got fixed when i restarted remix. sorry for this. i am able to compile now.

**Comment by EngrPips** [comment]
Awesome, I am glad you figured it out. Keep crushing it.
